metabase/README.md

```diff
@@ -0,0 +1,29 @@
+Metabase installation.
+
+Install Metabase from the Helm `stable` repository using Helm.
+
+```
+helm repo add stable https://kubernetes-charts.storage.googleapis.com
```

Comment:
The user has helm in their local folder from when we added `k3sup`, can you add this at the top please?

```
export PATH=$PATH:$HOME/.k3sup/bin/
```